scripts/main.js

```diff
@@ -1,3 +1,11 @@
+import { getAttractions } from "./attractions/AttractionProvider.js";
+import { getEateries } from "./eateries/EateryProvider.js";
 import {sendDropDownToDom} from "./nav/dropDown.js";
+import { getParks } from "./parks/ParkProvider.js";
+import { getWeather } from "./weather/WeatherProvider.js";
 
+getParks()
```

Comment:
These get functions should be called by the function that are writing to the DOM and not main.